zcutil/afl/zcash-wrapper

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -ex -o pipefail
+
+export ARGS=$@
+
+instrument=(
+"\/zcash\/src$"
```

Comment:
This assumes you've cloned the repo into a directory named `zcash` which may not be the case.